File: app/src/main/java/com/di/penopllast/vklikesremover/data/api/VkApi.java
package com.di.penopllast.vklikesremover.data.api;

import com.di.penopllast.vklikesremover.entity.ResponseFaveGetPosts;
import com.di.penopllast.vklikesremover.entity.RootFaveGetPost;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface VkApi {

    @GET("/method/fave.getPosts")
    Call<RootFaveGetPost> getLikes(@Query("offset") int offset, @Query("count") int count,
                                   @Query("extended") int extended,
                                   @Query("access_token") String token,
                                   @Query("v") String version);
}

File: app/src/main/java/com/di/penopllast/vklikesremover/data/api/VkApi.kt
package com.di.penopllast.vklikesremover.data.api

import com.di.penopllast.vklikesremover.entity.like.RootFaveGetPost
import com.di.penopllast.vklikesremover.entity.user.RootUser

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface VkApi {

    @GET("/method/fave.getPosts")
    fun getLikes(@Query("offset") offset: Int, @Query("count") count: Int,
                 @Query("extended") extended: Int, @Query("access_token") token: String,
                 @Query("v") version: String): Call<RootFaveGetPost>

    @GET("/method/users.get")
    fun getUser(@Query("access_token") token: String, @Query("user_ids") userId: Int,
                @Query("fields") field: String, @Query("name_case") nameCase: String,
                @Query("version") version: String): Call<RootUser>
}
